Add countersunk bolts and warning about use of Loctite on plastic (#431)
gamemanual0/gm0: source/docs/hardware-components/fastener-guide.rst

These bolts have a domed head, in contrast to the flat top of a standard socket bolt.

**Countersunk Head Bolts**

.. figure:: images/countersunk-head.jpg
   :alt: picture of a countersunk head bolt
   :width: 150px

> These bolts have a flat head, these can be used in countersunk holes to achieve a flush surface finish.

Reviewer: ```suggestion
These bolts have a cone-shaped head, and can be used in countersunk (cone-shaped) holes to achieve a flush surface finish.
```

      .. note:: It is highly recommended that teams use Loctite on all motor and :term:`servo <Servo>` mounts, as well as any mechanism prone to vibration.

>       .. danger:: Blue or red Loctite should not be used with brittle plastics such as polycarbonate. This is known to cause the plastic to crack. Instead, either use a `cyanoacrolate based glue <https://www.loctiteproducts.com/products/central-pdp.html/loctite-super-pro-gel/SAP_0201DGL029Z5.html>`_ or use a `screw with a nylon patch <https://www.mcmaster.com/products/screws/socket-head-screws~/18-8-stainless-steel-thread-locking-socket-head-screws/>`_ as this will not damage the plastic.
>       .. danger:: **THE BOTTLE COLOR AND THE FLUID COLOR ARE REVERSED.** When we refer to the "color", we mean the fluid color. Blue loctite usually comes in a red bottle.

Reviewer: ```suggestion
      .. danger:: 
         
         Blue or red Loctite should not be used with brittle plastics such as polycarbonate. This is known to cause the plastic to crack. Instead, either use a `cyanoacrolate based glue <https://www.loctiteproducts.com/products/central-pdp.html/loctite-super-pro-gel/SAP_0201DGL029Z5.html>`_ or use a `screw with a nylon patch <https://www.mcmaster.com/products/screws/socket-head-screws~/18-8-stainless-steel-thread-locking-socket-head-screws/>`_ as this will not damage the plastic.
         
         Also note: **THE BOTTLE COLOR AND THE FLUID COLOR ARE REVERSED.** When we refer to the "color", we mean the fluid color. Blue loctite usually comes in a red bottle.
```

      .. note:: It is highly recommended that teams use Loctite on all motor and :term:`servo <Servo>` mounts, as well as any mechanism prone to vibration.

>       .. danger:: 
>          
>          Blue or red Loctite should not be used with brittle plastics such as polycarbonate. This is known to cause the plastic to crack. Instead, either use a `cyanoacrolate based glue <https://www.loctiteproducts.com/products/central-pdp.html/loctite-super-pro-gel/SAP_0201DGL029Z5.html>`_ or use a `screw with a nylon patch <https://www.mcmaster.com/products/screws/socket-head-screws~/18-8-stainless-steel-thread-locking-socket-head-screws/>`_ as this will not damage the plastic.
>          
>          Also note: **THE BOTTLE COLOR AND THE FLUID COLOR ARE REVERSED.** When we refer to the "color", we mean the fluid color. Blue loctite usually comes in a red bottle.

Reviewer: ```suggestion
      .. danger::

         Blue or red Loctite should not be used with brittle plastics such as polycarbonate. This is known to cause the plastic to crack. Instead, either use a `cyanoacrolate based glue <https://www.loctiteproducts.com/products/central-pdp.html/loctite-super-pro-gel/SAP_0201DGL029Z5.html>`_ or use a `screw with a nylon patch <https://www.mcmaster.com/products/screws/socket-head-screws~/18-8-stainless-steel-thread-locking-socket-head-screws/>`_ as this will not damage the plastic.

         Also note: **THE BOTTLE COLOR AND THE FLUID COLOR ARE REVERSED.** When we refer to the "color", we mean the fluid color. Blue loctite usually comes in a red bottle.
```
